### PR TITLE
Bare implementation of event API

### LINKED ISSUE #39: Design Corvette API

### PR DESCRIPTION
**Pull request by eseiker**
closes #39

### CODE REVIEW

File: api.ts [hunk 1]
```diff
@@ -157,11 +147,5 @@ export async function api(
...
 
 if (import.meta.main) {
-  runWithPrisma((prisma) =>
-    Promise.resolve({
-      runningPromise: runWithAmqp((amqpConnection) =>
-        api(prisma, amqpConnection)
-      ),
-    })
-  );
+  runWithPrisma((prisma) => api(prisma));
```
Comment (tkiapril) [inline api.ts:150]:
```suggestion
  runWithPrisma(api);
```

File: responseUtil.ts
Comment (tkiapril) [inline responseUtil.ts:1]: What do you think about changing the name of this file to EventResponse.ts?

**Review by tkiapril — DISMISSED** [review]
Some minor comments.

File: responseUtil.ts
Comment (eseiker) [reply responseUtil.ts:1]: Sure, could you suggest any idea?

**Review by tkiapril — APPROVED** [review]

File: emitter.ts [hunk 2]
```diff
@@ -33,6 +31,7 @@ import {
   runWithPrisma,
 } from "./runHelpers.ts";
 import { uint8ArrayEquals } from "./uint8ArrayUtils.ts";
+import { serializeEventResponse } from "./responseUtil.ts";
```
Comment (tkiapril) [inline emitter.ts:34]:
```suggestion
import { serializeEventResponse } from "./EventResponse.ts";
```

**Review by tkiapril — CHANGES_REQUESTED** [review]

**Review by tkiapril — APPROVED** [review]

**Pull request merged**